src/main/java/frc/robot/commands/Arm/ScoreInTrap.java

```diff
@@ -25,6 +25,7 @@ public ScoreInTrap(PizzaBoxSubsystem pizzaBoxSubsystem, ArmSubsystem armSubsyste
         new StopNoteContainerMotor(pizzaBoxSubsystem),
         new SpinToArmAngle(armSubsystem, 135)
     );
+    pizzaBoxSubsystem.hasNote = false;
```

Comment:
Should this be in initialize, or the first 'InstantCommand' so it gets set false every time?